utils/helpers.js

```diff
@@ -0,0 +1,5 @@
+function preprocessOwaspReport (inputFile) {
```

Comment:
You could use an arrow function for consistency here.

Reply:
the function was replaced by an arrow one.